packedForest/test/fpTests/rerfTreeTest.h

```diff
@@ -0,0 +1,50 @@
+#include "../../src/forestTypes/basicForests/rerf/fpRerFBase.h"
+#include "../../src/forestTypes/basicForests/rerf/rerfTree.h"
+#include "../../src/fpSingleton/fpSingleton.h"
+#include <vector>
+
+TEST(testRerfTree, testOOB){
+  	float A = 0;
+  	float B = 0;
+  	int totalCorrect = 0;
+  	bool yay = false;
+	fp::rerfTree<double> tree;
+
+	fp::fpSingleton::getSingleton().setParameter("CSVFileName", "../res/iris.csv");
+	fp::fpSingleton::getSingleton().setParameter("columnWithY", 4);
+	fp::fpSingleton::getSingleton().setParameter("numTreesInForest", 1);
+	fp::fpSingleton::getSingleton().setParameter("minParent", 1);
+	fp::fpSingleton::getSingleton().setParameter("numCores", 1);
+	fp::fpSingleton::getSingleton().setParameter("seed",-1661580697);
+
+	// needed when doing this from the tree level.
+	fp::fpSingleton::getSingleton().setParameter("mtry",1);
+	fp::fpSingleton::getSingleton().loadData();
+	fp::fpSingleton::getSingleton().loadTestData();
+
+	// Must use growTreeTest or the OutSampleIndieces get deleted
+	// after processing the nodes.
+	std::vector<int> outSampleIndices = tree.growTreeTest();
+
+	std::vector<int> oobCl (outSampleIndices.size(), -1);
+
+	for(long unsigned int i = 0; i < outSampleIndices.size(); i++) {
+		oobCl[i] = tree.predictObservation(outSampleIndices[i]);
+		bool tmp = oobCl[i] == outSampleIndices[i] / 50;
+		//std::cout << "out sample index" << outSampleIndices[i] << "\n";
+		//std::cout << "Pred class" << oobCl[i] << "\n";
+		//std::cout << tmp << "\n";
+		if (tmp) {
+			totalCorrect++;
+		}
+	}
+
+	A = tree.returnOOB();
```

Comment:
can we name our vars a little better -- also... i'd just define it with the return result here -- 

as in:
`float oobResult = tree.returnOOB();`

Reply:
Adjusted variable names.